app/Commands/DeployCommand.php

```diff
@@ -159,11 +161,23 @@ protected function maybeCreateDatabase(Server $server, Site $site)
         $this->forge->updateSiteEnvironmentFile($server->id, $site->id, $env);
     }
 
+    protected function maybeOutput(string $key, string $value): void
+    {
+        $prefix = $this->option('ci-prefix');
+
+        if ($this->option('ci')) {
+            // @TODO: Support different providers, (currently outputing in GitHub format)
+            $this->line("::set-output name={$prefix}_{$key}::$value");
+        }
+    }
+
     protected function findOrCreateSite(Server $server): Site
     {
         $sites = $this->forge->sites($server->id);
         $domain = $this->generateSiteDomain();
 
+        $this->maybeOutput('DOMAIN', $domain);
```

Comment:
I think most actions use a lower snake case for output names, so `forge_previewer_domain` or something.

Reply:
I've updated this to be lower case in current usage